### PR TITLE
 added a contact page in the menu bar options[P3]

### PR DESCRIPTION
**Pull request by anjalisoni3655**
Feature added:
1.Added a contact us page with three  buttons:

- Website - redirects to the link of the company.

- Phone number - user can dial to the company
- email - user can mail the company

 Testing: Tested on local devices and using flutter test command by adding unit test cases.
Result: All tests passed.

menu

contact

### CODE REVIEW

**Comment by anjalisoni3655** [comment]
@googlebot I consent.

**Comment by anjalisoni3655** [comment]
@googlebot I consent

File: home_insurance_app/lib/pages/contact.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import 'package:flutter/material.dart';
+
+class Contact extends StatelessWidget {
+  static const String id = 'contact';
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      home: Scaffold(
+        backgroundColor: new Color(0xFFE57373),
```
[the remark this answers is not in the document]
Comment (anjalisoni3655) [reply home_insurance_app/lib/pages/contact.dart:11]: have made a components folder, and inside it a  contants.dart file, where I will putting all the css type of code

**Comment by anjalisoni3655** [comment]
@googlebot I consent.

**Comment by khushi2305** [comment]
@googlebot I consent.

File: home_insurance_app/lib/components/constants.dart [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  fontWeight: FontWeight.bold,
+);
+
+/// Normal TextStyle for various usages.
+/// [color: Black, fontFamily: Source Sans Pro, size: 20.0]
+const kNormalTextStyle = TextStyle(
+  color: Colors.black,
+  fontFamily: 'Source Sans Pro',
+  fontSize: 20.0,
+);
```
Comment (akashagarwal) [inline home_insurance_app/lib/components/constants.dart:26]: change file name to something better, probably css.dart or something
constants file name is usually used for other code constant like static strings

File: home_insurance_app/lib/pages/profile.dart [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+              child: Divider(
+                color: Colors.black,
+              ),
+            ),
+            Card(
+                margin: EdgeInsets.symmetric(vertical: 10.0, horizontal: 25.0),
+                child: ListTile(
+                  leading: Icon(
+                    Icons.phone,
+                    color: Colors.black,
```
Comment (akashagarwal) [inline home_insurance_app/lib/pages/profile.dart:41]: all colors can be part of common styles

File: home_insurance_app/lib/components/constants.dart (hunk 2 above)
Comment (anjalisoni3655) [reply home_insurance_app/lib/components/constants.dart]: done.

**Comment by dreamplay** [comment]
can you add a screenshot for where Contact us tab is visible in App

**Pull request merged**